src/models/team.js

```diff
@@ -1,4 +1,11 @@
 const { Model } = require('sequelize');
+var en = require('nanoid-good/locale/en');
+var fr = require('nanoid-good/locale/fr');
+var customAlphabet = require('nanoid-good').customAlphabet(en, fr);
+
+// Based on documentation here https://github.com/ai/nanoid
+const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
Altered the alphabet according to the previous comment and also added support for french swear word blocking.